e2e-tests/proxy-protocol/run

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+cluster="proxy-protocol"
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+prepare_config() {
+	nodes=$(kubectl get nodes | grep -v "NAME" | awk '{print $1}')
+	if [ $(printf "${nodes}\n" | wc -l) -lt 3 ]; then
+		echo "3 nodes are expected for this test to run."
+		exit 1
+	fi
+	haproxy_node=$(echo "${nodes}" | head -n1)
+	pxc_node=$(echo "${nodes}" | head -n2 | tail -n1)
+	client_node=$(echo "${nodes}" | head -n3 | tail -n1)
+	cat ${test_dir}/conf/${cluster}.yml \
+		| $sed -e "s#kubernetes.io/hostname:.*-node1\$#kubernetes.io/hostname: ${haproxy_node}#" \
+		| $sed -e "s#kubernetes.io/hostname:.*-node2\$#kubernetes.io/hostname: ${pxc_node}#" >${tmp_dir}/${cluster}.yml
+	cat ${test_dir}/conf/client.yml | $sed -e "s#kubernetes.io/hostname:.*-node3\$#kubernetes.io/hostname: ${client_node}#" >${tmp_dir}/client.yml
+}
+
+spinup_pxc() {
```

Comment:
Because it tries to connect directly and then stalls because it doesn't work. We bind mysql to `0.0.0.0` and specify `proxy_protocol_networks=*` so mysql sees APP IP, but direct connection doesn't work.
Check the table at the end of this blogpost: https://www.percona.com/blog/2015/10/15/proxy-protocol-percona-xtradb-cluster-quick-guide/
to see which combinations work and which don't.

Reply:
you can add possibility to pass port for this function and use 33062 port to connect to DB

Reply:
Thanks, added! Let's see how the tests go.